### PR TITLE
rwx: implement test_encrypted_rwx_volume

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn/longhorn#7045
Longhorn/longhorn#7046

### CODE REVIEW

File: manager/integration/tests/test_rwx.py [hunk 1]
```diff
@@ -634,3 +638,48 @@ def test_rwx_offline_expansion(client, core_api, pvc, make_deployment_with_pvc):
...
+
+    deployment = make_deployment_with_pvc(
+        'pvc-deployment-with-encrypted-rwx-volume', pvc_name, replicas=3)
+
+    apps_api = get_apps_api_client()
+    create_and_wait_deployment(apps_api, deployment)
+
+    # Clean up deployment and volume
+    delete_and_wait_deployment(apps_api, deployment["metadata"]["name"])
+    delete_and_wait_pvc(core_api, pvc_name)
```
Comment (khushboo-rancher) [inline manager/integration/tests/test_rwx.py:685]: Clean up is handled in the fixtures.

**Review by khushboo-rancher — COMMENTED** [review]
LGTM, just a minor comment.

File: manager/integration/tests/test_rwx.py (hunk 1 above)
Comment (yangchiu) [reply manager/integration/tests/test_rwx.py:685]: @derekbit Would you like to improve this? Or just merge directly?

**Review by yangchiu — APPROVED** [review]
LGTM

**Pull request merged**